README.md

```diff
@@ -86,3 +86,71 @@ npm run serve
 ## Example
 The following example showcases ScaffoldVuer in action: https://mapcore-demo.org/current/scaffoldvuer/
 
+## Scaffoldvuer release process:
+
+
+#### Publishing 
+
+1. Update the package number in [package.json](https://github.com/ABI-Software/scaffoldvuer/blob/main/package.json)
+2. Bundles for scaffoldvuer can be built by using 
+```
+npm run build-bundle
+```
+
+### Versioning
+
+Scaffoldvuer uses [semantic versioning](https://semver.org/).
+
+If any changes are experimental or parallel to the main development, [npm dist tags](https://docs.npmjs.com/cli/v9/commands/npm-dist-tag) are used to keep this version from being the default when `npm install @abi-software/scaffoldvuer` is used.
+
+Dist tags can be used like so:
+```
+npm publish --tag <tag>
+```
+
+`<tag>` is the dist tag to add. In example: `npm publish --tag beta`
+
+This builds the package to be digested by other vue packages by using
+```
+import { ScaffoldVuer } from "@abi-software/scaffoldvuer/src/components/index.js";
```

Comment:
We are building the mapintegratedvuer with the source directly but it should normally be done using 
`import { ScaffoldVuer } from "@abi-software/scaffoldvuer;` in other cases. 

css should be imported  as well when including scaffoldvuer using `import { ScaffoldVuer } from "@abi-software/scaffoldvuer;`.